Packs/CommonScripts/Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -191,11 +190,11 @@ def PtypString8(data_value):
     def PtypString(data_value):
         if data_value:
             try:
+                data_value = data_value.decode("utf-16-le", errors="ignore").replace('\x00', '')
+            except UnicodeDecodeError:
                 res = chardet.detect(data_value)
                 enc = res['encoding'] or 'ascii'  # in rare cases chardet fails to detect and return None as encoding
```

Comment:
I don't remember exactly. But I think there was a case where ascii was the encoding.
What is the value we get from `res['encoding']` ?

Reply:
@anara123 This should detect the encoding of the given byte string.
But in our case it returns `ASCII` and it does not encode it well, and also in the case of an English string it returns `ASCII`, and yet when I change it to `utf-16-le` it encodes it well.